src/main/java/seedu/agendum/model/ModelManager.java

```diff
@@ -26,29 +32,33 @@
 
     private final ToDoList toDoList;
     private final FilteredList<Task> filteredTasks;
+    private final Config config;
 
     /**
      * Initializes a ModelManager with the given ToDoList
      * ToDoList and its variables should not be null
      */
-    public ModelManager(ToDoList src, UserPrefs userPrefs) {
+    public ModelManager(ToDoList src, UserPrefs userPrefs, Config config) {
         super();
         assert src != null;
         assert userPrefs != null;
+        assert config != null;
 
         logger.fine("Initializing with to do list: " + src + " and user prefs " + userPrefs);
 
         toDoList = new ToDoList(src);
         filteredTasks = new FilteredList<>(toDoList.getTasks());
+        this.config = config;
```

Comment:
I wanted to ask if it is possible to use Storage directly instead of going through Model.
But I think to load from an existing file, Model will most probably need info from config/storage.